cypress/support/e2e.ts

```diff
@@ -5,8 +5,28 @@ import './commands';
 import { createGlobalOrganization, createGlobalProject } from './global-project';
 
 before(function () {
+  let splitLocalhost;
   const devBaseUrlPort = Cypress.config().baseUrl?.split(':').slice(-1).toString();
-  if (devBaseUrlPort === '4101') {
+  const baseUrl = Cypress.config().baseUrl;
+  let localhost;
+  // Assuming baseUrl = "https://localhost.com"
```

Comment:
Could you add a brief comment here to explain this code block please?

Reply:
@vidyanambiar I am in the process of changing this code block. we may not need it. pending checks.